ads/_config.js

```diff
@@ -941,6 +941,10 @@ export const adConfig = {
     consentHandlingOverride: true,
   },
 
+  'torimochi': {
+    renderStartImplemented: true,
```

Comment:
Hello! 

So everything looks good, and everything is working:

![screen shot 2019-02-25 at 7 15 33 pm](https://user-images.githubusercontent.com/1448289/53385009-d0286600-3931-11e9-970c-88519712c920.png)

However, it seems like the `renderStart` is not working. I set some logging for `renderStart_()` in `amp-ad-xorigin-iframe-handler.js`. But it never seems to get called. 

Does your vendor implement `renderStart`? If not, could you please implement this? It helps us serve ads as fast as possible to our users, and gives us some big performance gains.

Let me know what you think. Thank you! 😄